GeoLib/Polygon.cpp

```diff
@@ -38,6 +38,25 @@ Polygon::Polygon(Polygon const& other)
 	}
 }
 
+Polygon& Polygon::operator=(Polygon const& other)
+{
+	if (other._ply_pnts != _ply_pnts) {
+		ERR(
+		    "Polygon::operator=: polygon other and this polygon are not based "
+		    "on the same point vector. Aborting");
```

Comment:
The polygons do not store the points but have a reference to point vector. So depending on how the points are managed all can go wrong here w/o the if. The details need to be explained anyway; Tom?